File: app/src/main/java/com/icesoft/msdb/android/model/RacetrackLayout.java
package com.icesoft.msdb.android.model;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper=false)
public class RacetrackLayout extends BaseModel {

    private Long id;
    private String name;
    private Integer length;
    private Integer yearFirstUse;
    private Boolean active;
    @JsonProperty("layoutImageUrl")
    private String layoutUrl;
    private Racetrack racetrack;
}

File: app/src/main/java/com/icesoft/msdb/android/model/RacetrackLayout.kt
package com.icesoft.msdb.android.model

data class RacetrackLayout(
    val id: Long? = null,
    val name: String? = null,
    val length: Int? = null,
    val yearFirstUse: Int? = null,
    val active: Boolean? = null,
    val layoutImageUrl: String? = null,
    val racetrack: Racetrack? = null
): BaseModel()
